CHANGELOG.rst

```diff
@@ -2,6 +2,29 @@
 Changelog
 =========
 
+Version 16.0
+=============
+
+* Fixed Deneb Readout errors to closer resemble reality. Fidelities were reported as errors. `#125 <https://github.com/iqm-finland/iqm-client/pull/125>`_
+* Refactored :meth:`IQMBackend.create_run_request` to improve user experience when using IQM specific run options.
+* Updated the documentation for using additional run options with IQM backends.
+* :meth:`IQMBackendBase.qubit_name_to_index` and :meth:`IQMBackendBase.index_to_qubit_name` now raises an error when using an invalid qubit name or index, rather than returning None.
+* Introduction of `IQMBackendBase.physical_target` and `IQMBackendBase.fake_target` to represent the physical quantum architectures and a Qiskit-compatible version, respectively.        
+* Added support for ``qiskit == 1.2`` and ``qiskit-aer == 1.5``.
+* Moved the circuit serialization logic from :class:`IQMProvider` to :mod:`iqm.qiskit_iqm.qiskit_to_iqm`.
+* Refactoring of the Qiskit transpiler:
+    * The Qiskit transpiler now automatically uses the :class:`IQMOptimizeSingleQubitGates` pass to optimize single-qubit gates if the `optimization_level >= 0`.
+    * You can now use the native Qiskit :meth:`transpile` method to transpile a circuit to the IQM Deneb backend as long as your circuit does not contain any resonators.
+    * There are many new transpiler plugins available that you can use as the `scheduling_method` argument in Qiskit's :meth:`transpile` method. You can find them in following the `Qiskit documentation <https://docs.quantum.ibm.com/guides/transpiler-plugins>`_. 
```

Comment:
Only `scheduling_method`? What about the other *_method args?

Reply:
The plugins are only registered as `scheduling_method` because they make assumptions about how much the circuit has been transpiled already. You can still combine them with the other transpiler args. 
You could make our own optimization plugin or routing plugin, but that would be exposed to the user as a string making them overwrite the existing transpiler and making it impossible to augment it with an existing pass. So they would need to defined with an existing optimization/routing pass and we need a new class for each or combination of passes we want to support. That's not maintainable.